src/test/features/response/routes/partial-admission.ts

```diff
@@ -1,59 +0,0 @@
-import { expect } from 'chai'
-import * as request from 'supertest'
-import * as config from 'config'
-
-import { attachDefaultHooks } from '../../../routes/hooks'
-import '../../../routes/expectations'
-import { checkAuthorizationGuards } from './checks/authorization-check'
-
-import { Paths as ResponsePaths } from 'response/paths'
-
-import { app } from '../../../../main/app'
-
-import * as idamServiceMock from '../../../http-mocks/idam'
-import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
-import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
-
-import { checkCountyCourtJudgmentRequestedGuard } from './checks/ccj-requested-check'
-import { checkNotDefendantInCaseGuard } from './checks/not-defendant-in-case-check'
-
-const cookieName: string = config.get<string>('session.cookieName')
-const pagePath = ResponsePaths.partialAdmissionPage.evaluateUri({ externalId: claimStoreServiceMock.sampleClaimObj.externalId })
-
-describe('Defendant response: partial admission page', () => {
```

Comment:
why is this removed and no other tests added?

Reply:
I see now this is because `ResponsePaths.partialAdmissionPage` handoff page is removed and won't be used anymore. The confirmation page is reused now when response is submitted to claim-store. Only test case we need to add is in `check-and-send.ts` test file to reflect new redirect behaviour.